### DISCUSSION TITLE
How to create a custom schema in Airbyte?

### DISCUSSION THREAD

**Discussion by komalm** [airbytehq/airbyte#32812] [Connector Questions]
We wanted to use Airbyte to sync data from a **Cassandra** database to a **Postgres** database. However, after some inquiry, we discovered that "_**The Cassandra source is not currently available in Airbyte**_." So we came up with a fresh solution: _**"We export CSV from Cassandra and use that."**_ (As a custom schema)

But we don't know how to use CSV files as a source right now.

Please guide us on this.

**Comment by pgrant87** [comment]
Hi @komalm 

Currently, we don't have a Cassandra source but this is something that has been requested in https://github.com/airbytehq/airbyte/discussions/32736 I can't confirm that this will get developed but I'll add this case there and tag you to try to give that as much traction as possible.

In the meantime, if you have your data in a .csv, you could use the file connector source but please note this can't be used with Airbyte Cloud: https://docs.airbyte.com/integrations/sources/file

If you are an Airbyte Cloud user, you could put your .csv in a google sheet and use the Google Sheets connector: https://docs.airbyte.com/integrations/sources/google-sheets

Unless your .csv is being updated somehow neither of these provide an ideal solution so if you're feeling adventurous, developing your own Cassandra source might be the best option here using one of our connector development frameworks: https://docs.airbyte.com/connector-development/

[a comment by an automated account is not shown]
